frame/esg/src/lib.rs

```diff
@@ -5,31 +5,35 @@ pub use pallet::*;
 #[cfg(test)]
 mod mock;
 
-#[cfg(feature = "runtime-benchmarks")]
-pub mod benchmarking;
 #[cfg(test)]
+
 pub mod tests;
 
+pub mod traits;
+
 pub mod weights;
 
-pub mod traits;
+#[cfg(feature = "runtime-benchmarks")]
+pub mod benchmarking;
 
 pub trait Sustainability<AccountId> {
-	fn get_score_of(company: AccountId) -> u16;
+	fn get_score_of(company: firechain_runtime_core_primitives::opaque::AccountId) -> u16;
 }
 
 #[frame_support::pallet]
 pub mod pallet {
-	use crate::{traits::ERScoresTrait, weights::WeightInfo};
-	use bs58;
-	use core::num::IntErrorKind;
 	use frame_support::{
-		pallet_prelude::{DispatchResult, *},
 		WeakBoundedVec,
+		pallet_prelude::{DispatchResult, *},
 	};
-	use frame_system::pallet_prelude::*;
-	use serde_json::Value;
 	use sp_std::vec::Vec;
+	use serde_json::Value;
+	use core::num::IntErrorKind;
+	use fp_account::AccountId20;
+	use frame_system::pallet_prelude::*;
+	use crate::{traits::ERScoresTrait, weights::WeightInfo};
+	use firechain_runtime_core_primitives::opaque::AccountId as AccIdEth;
```

Comment:
Why dont we use `T::Account` from `frame_system` . 
in frame_system::Config 
`type AccountId = AccountId` -> `AccountId` that we take from `firechain_runtime_core_primitives::opaque::AccountId`

Reply:
because the default type AccountId is internally of AccountId32

Reply:
I am still using AccIdEth as generic type wherever it needs to be